rmgpy/molecule/drawTest.py

```diff
@@ -126,6 +126,28 @@ def test_draw_non_standard_bonds(self):
         surface, cr, (xoff, yoff, width, height) = self.drawer.draw(hybrid, file_format='pdf')
         self.assertIsInstance(surface, PDFSurface)
 
+    def test_draw_hydrogen_bond_adsorbate(self):
+
+        molecule = Molecule().from_adjacency_list("""
+1  O u0 p3 c-1 {2,S} {10,H}
+2  N u0 p0 c+1 {1,S} {3,D} {4,S}
+3  O u0 p2 c0 {2,D}
+4  O u0 p2 c0 {2,S} {7,S}
+5  N u0 p1 c0 {6,S} {8,S} {9,S} {7,H}
+6  O u0 p2 c0 {5,S} {10,S}
+7  H u0 p0 c0 {4,S} {5,H}
+8  H u0 p0 c0 {5,S}
+9  H u0 p0 c0 {5,S}
+10 H u0 p0 c0 {6,S} {1,H}
+11 X u0 p0 c0
+        """
+        )
+        try:
+            from cairocffi import PDFSurface
```

Comment:
Are `cairocffi` and `cairo` OS-dependent? If not, can we assume one of them should be present?

Reply:
I don't know - I just copied boilerplate from all the other drawing tests. Changing them all is out of scope of this PR.